ROOTFS/etc/waggle/sanity/warning/check_hanwha_camera.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
```

Comment:
this file must end in .test in order to be auto run

should this be fatal?  if a camera doent work isn't it fatal?

Reply:
Changed to .test and move it in `fatal`. I initially thought it is not fatal to the system, but for plugins.